src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+/* .stars-container {
+  display: flex;
+  flex-direction: column;
+} */
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  display: inline;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.star--active {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
Remove this class and let's specify active stars in another way.

Reply:
Thank you for your attitude, it is very inspiring! ))